pkgs/applications/misc/lutris/chrootenv.nix

```diff
@@ -113,5 +113,13 @@ in buildFHSUserEnv {
     mkdir -p $out/share
     ln -sf ${lutris-unwrapped}/share/applications $out/share
     ln -sf ${lutris-unwrapped}/share/icons $out/share
+    ln -s $out/bin/lutris-${lutris-unwrapped.version} $out/bin/lutris
```

Comment:
This is not a metadata change. Please move it to separate commit.

Reply:
this